packages/org-unit-dialog/src/UserOrgUnitsPanel.js

```diff
@@ -9,39 +10,50 @@ import StopIcon from '@material-ui/icons/Stop';
 import userOrgUnits from './userOrgUnits';
 import styles from './styles/UserOrgUnitsPanel';
 
-const UserOrgUnitsPanel = props => (
-    <div style={styles.container}>
-        <Grid
-            container
-            direction="row"
-            alignItems="center"
-        >
-            {userOrgUnits.map(orgUnitType => (
-                <Grid
-                    key={orgUnitType.id}
-                    style={styles.gridItem}
-                    item
-                >
-                    <Checkbox
-                        checked={props.userOrgUnits.some(ouType => ouType.id === orgUnitType.id)}
-                        onChange={props.handleUserOrgUnitClick}
-                        inputProps={{
-                            id: orgUnitType.id,
-                            name: orgUnitType.id,
-                        }}
-                        icon={<CheckBoxOutlineBlankIcon style={styles.checkbox} />}
-                        checkedIcon={<CheckBoxIcon style={styles.checkboxChecked} />}
-                        color={props.checkboxColor}
-                    />
-                    <InputLabel htmlFor={orgUnitType.id}>
-                        <StopIcon style={styles.stopIcon} />
-                        <span style={styles.text}>{orgUnitType.displayName}</span>
-                    </InputLabel>
-                </Grid>
-            ))}
-        </Grid>
-    </div>
-);
+const UserOrgUnitsPanel = props => {
+    const orgUnitNames = {
+        USER_ORGUNIT: i18n.t('User organisation unit'),
+        USER_ORGUNIT_CHILDREN: i18n.t('User sub-units'),
+        USER_ORGUNIT_GRANDCHILDREN: i18n.t('User sub-x2-units')
+    };
+
+    return (
+        <div style={styles.container}>
+            <Grid
+                container
+                direction="row"
+                alignItems="center"
+            >
+                {userOrgUnits.map(orgUnitType => {
+                    const orgUnitName = orgUnitNames[orgUnitType.id];
```

Comment:
if the value for the `const` is moved directly to line 49 (only used in 1 place anyway) the `return` statement can be removed to make it a bit cleaner.

Reply:
👍